pkg/connector/groups.go

```diff
@@ -95,6 +94,7 @@ func (g *groupBuilder) List(ctx context.Context, parentResourceID *v2.ResourceId
 
 	groups, total, _, err := g.client.ListGroups(
 		ctx,
+		workspaceId,
```

Comment:
it is okay if `workspaceId` is empty string?

Reply:
Yes. Databricks has two APIs: one for whole databricks accounts and one for just a workspace in an account. If we're configured with `--workspace-tokens` and `--workspace-ids`, we use the workspace API. Otherwise we use the account API.

See https://github.com/ConductorOne/baton-databricks/blob/393a768707b206e28fb17377e14e869173e4a9a9/pkg/databricks/client.go#L121 and similar functions for examples of how we treat empty workspace IDs.

Reply:
I'm satisfied with the explanation :)